src/ott/tools/gaussian_mixture/gaussian_mixture.py

```diff
@@ -141,26 +141,32 @@ def from_points_and_assignment_probs(
 
   @property
   def dtype(self):
+    """Get the dtype of the GMM parameters."""
```

Comment:
If flake8 won't give you errors, remove `Get the` in the description of these, like `Number of dimensions of the GMM parameters.`, etc.